src/langcheck/metrics/eval_clients/_openai.py

```diff
@@ -114,11 +121,13 @@ def get_text_responses(
             A list of responses to the prompts. The responses can be None if the
             evaluation fails.
         """
-        config = {"model": "gpt-3.5-turbo", "seed": 123}
+        config = {"model": "gpt-3.5-turbo"}
```

Comment:
Now `AzureOpenAIEvalClient` only set the `model` parameter in the `get_score`. So when `get_text_responses` is directly called from `augment_from_templateaugment_from_template`, `"gpt-3.5-turbo"` is always used as the model name even if you set one.

I think it is fine to set the param in the `__init__` as the following:

```python
        self._text_model_name = text_model_name
        self._embedding_model_name = embedding_model_name

        self._openai_args = openai_args or {}
        if self._text_model_name:
            self._openai_args["model"] = self._text_model_name
```

Then maybe we can remove `get_score` of `AzureOpenAIEvalClient`. Could you work on that?